src/main/java/fr/techad/edc/popover/internal/swing/EdcSwingHelpImpl.java

```diff
@@ -40,12 +42,24 @@ public JComponent createComponent(String mainKey, String subKey, String iconPath
         HelpConfiguration helpConfiguration = getHelpConfiguration();
         String languageCode = helpConfiguration.getLanguageCode();
 
-        JComponent component = contextualComponentBuilder.setKeys(mainKey, subKey, languageCode)
+        try {
+            enableContextItem = edcClient.getContextItem(mainKey, subKey, languageCode) != null;
+        } catch (InvalidUrlException | IOException e) {
```

Comment:
ok, so that is the enableContextItem in this case?
Perhaps, you have to force the value: false?